packages/sanity/src/core/form/inputs/PortableText/hooks/useTrackFocusPath.tsx

```diff
@@ -27,6 +27,13 @@ export function useTrackFocusPath(props: Props): void {
       .filter((item) => item.member.open)
       .sort((a, b) => b.member.item.path.length - a.member.item.path.length)[0]
     if (memberItem && memberItem.elementRef?.current) {
+      // Don't do anything if we have already "closed" a textBlock (see below) or have a object open for editing.
+      const isTextBlockClosed = memberItem.kind === 'textBlock' && !memberItem.member.open
+      const isObjectOpen = memberItem.kind !== 'textBlock' && memberItem.member.open
```

Comment:
Will this be true with the case where you only have an object "selected" in the editor, but not having focus inside?

Ref. what we talked about offline @bjoerge